code/planning/launch/planning.launch

```diff
@@ -1,4 +1,4 @@
-<!-- TODO: Insert components of component -->
+<!-- TODO: Insert components of component-->
 <launch>
     <node pkg="planning" type="dev_global_route.py" name="DevGlobalRoute" output="screen">
```

Comment:
Why is the static route from a file still used here? How could this be tested with the leaderboard?

Reply:
in our dev environment the leaderboard isnt active therefore no global plan is being published therfore the route that is being published by the leaderboard was saved and is now published by this node to simmulate it and use it in dev environment as well